### ISSUE TITLE
Only event organizer see the event 

### ISSUE THREAD

**Issue by hantt12** [OpenPaaS-Suite/esn-frontend-calendar#101]
Scenario 1:

1. Usera shares calendar X to userb (either by delegation or public share)

2. Usera creates event in calendar X

=> Userb does not see the above event

Scenario 2: 

1. Usera shares calendar X to userb (either by delegation or public share)
2. Userb creates event in calendar X
=> Usera does not see the above event

Scenario 3: 
1. Usera shares calendar X to userb (either by delegation or public share)
2. Usera/userb creates event in calendar X and add userc as attendee
=> Userc does not see the event

Scenario 4: 
1. Usera create an event in his calendar and invite user b
=> Userb does not see the event

To SUM UP: only people who create the event can see the event in his calendar
